src/os/bluestore/BlueStore.h

```diff
@@ -2061,6 +2063,10 @@ class BlueStore : public ObjectStore,
   void _set_alloc_sizes();
   void _set_blob_size();
   void _set_finisher_num();
+  void _update_osd_memory_options();
```

Comment:
nit: any rationales for having two different handlers for osd_memory_xxx config options changes?

Reply:
My rationale was that `osd_memory_target `would probably be modified more often than the other options based on the fact that `osd_memory_target` is marked as 'basic' and the other options are marked as either 'advanced' or 'dev'.  I therefore felt that if the user only modifies `osd_memory_target` option, there's no necessity to retrieve all other options like I do within `_update_osd_memory_options()`. But if you still think this is overkill, I can consolidate all options into `_update_osd_memory_options()`.

Reply:
Sure, I have consolidated all the options into one function (`_update_osd_memory_options()`). Thanks for the feedback.